risedev.yml

```diff
@@ -232,10 +232,10 @@ risedev:
 
 compose:
   risingwave: "ghcr.io/singularity-data/risingwave:latest"
-  compute-node: "ghcr.io/singularity-data/compute-node:latest"
-  meta-node: "ghcr.io/singularity-data/meta-node:latest"
-  compactor-node: "ghcr.io/singularity-data/compactor-node:latest"
-  frontend-node: "ghcr.io/singularity-data/frontend-node:latest"
+  # compute-node: "ghcr.io/singularity-data/compute-node:latest"
+  # meta-node: "ghcr.io/singularity-data/meta-node:latest"
+  # compactor-node: "ghcr.io/singularity-data/compactor-node:latest"
+  # frontend-node: "ghcr.io/singularity-data/frontend-node:latest"
```

Comment:
Why commented these lines? Do you intend to let users uncomment them?

Reply:
Yes, currently we don't expect users to use the single `meta-node` image. We can enable them later when we have version tags on images.

Maybe let's remove them instead of commenting out?

Reply:
Also risedev-compose will only produce docker-compose configuration on `risingwave` image instead of other single-component images.